airflow/config_templates/config.yml

```
@@ -1767,6 +1767,14 @@
       type: string
       example: ~
       default: "30"
+    - name: deactivate_stale_dags_interval
+      description: |
+        How often (in seconds) to check for stale DAGs (DAGs which are no longer present in
+        the expected files) which should be deactivated.
+      version_added: 2.3.0
```

```suggestion
      version_added: 2.2.5
```

BTW is it a feature or a bug?
Maybe the PR that added it should not be targeted to 2.2.5?

---

It's a bug fix. it was previously targeted for 2.3.0. Thanks for this